salt/modules/cp.py

```diff
@@ -8,6 +8,9 @@
 import os
 import logging
 import fnmatch
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
This is going to set the default encoding for the entire Python process to UTF-8 despite what the system might be configured for. I don't think we want to do this.